test/dryrun/context_vars/object_namespaced/policy.yaml

```diff
@@ -19,4 +19,4 @@ spec:
           name: '{{ .ObjectName }}'
           namespace: '{{ .ObjectNamespace }}'
           labels:
-            new-label: '{{ (fromYAML (index .Object.data "inventory.yaml")).appliance }}'
+            new-label: '{{ ne (printf "%s" .ObjectName) "inventory" | skipObject }}{{ (fromYAML (index .Object.data "inventory.yaml")).appliance }}'
```

Comment:
Note: Prevents parsing errors when iterating over config maps in the cluster that don't match the desired name and data. e.g. `kube-root-ca.crt`